util/freebsd_install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 util_dir=$(dirname "$0")
-pkg update
-pkg install -y \
-	git \
-	wget \
-	gmake \
-	gcc \
-	zip \
-	unzip \
-	avr-binutils \
-	avr-gcc \
-	avr-libc \
-	dfu-programmer \
-	dfu-util \
-	arm-none-eabi-gcc \
-	arm-none-eabi-binutils \
-	arm-none-eabi-newlib \
-	diffutils \
-	python3
-pip3 install -r ${util_dir}/../requirements.txt
+if [ $(id -u) = 0 ]
+then
```

Comment:
To match the style of other shell scripts.

```suggestion
if [ $(id -u) = 0 ]; then
```